_posts/2020-10-20-3.8.0-release-announcement.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp 3.8.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - announcement
+---
+
+The Pulp team are pleased to announce the release of Pulp 3.8.0!
+
+## Updates to Django settings
+
+* The Django `admin` site URL is now configurable using the new `ADMIN_SITE_URL` settings parameter [7637](https://pulp.plan.io/issues/7637).
+
+* As a follow up to [#7179](https://pulp.plan.io/issues/7179), which provided a default `DJANGO_SETTINGS_MODULE` variable, in this release, the default `DJANGO_SETTINGS_MODULE` variable is now set everywhere. This simplifies configuration because the **systemd** service no longer has to set this value [#7720](https://pulp.plan.io/issues/7720).
+
+## Allowed Content Checksums Validation
+
+In the previous release `ALLOWED_CONTENT_CHECKSUMS` variable was introduced so that users can specify and limit unwanted checksums in their environment.
+
+In this release, validation has been added so that if a user has Artifacts in their database and adds checksums to `ALLOWED_CONTENT_CHECKSUMS` that do not match the existing Artifacts, the error `ImproperlyConfigured` is thrown [#7487](https://pulp.plan.io/issues/7487).
+
+## Dynaconf Version Requirement
+
+Earlier versions of dynaconf proved buggy and error prone. In this release, the required dynaconf version is 3.1.2. [#7682](https://pulp.plan.io/issues/7682).
```

Comment:
Shall we be a bit softer to dynaconf? ;) 
Maybe something like this? `Earlier versions of dynaconf had some known issues. In this release,..`
Up to you. Just caught my eye :)

Reply:
Sincerest apologies to dynaconf ;)